modules/representation_management/app/controllers/representation_management/v0/accredited_individuals_controller.rb

```diff
@@ -0,0 +1,163 @@
+# frozen_string_literal: true
+
+module RepresentationManagement
+  module V0
+    class AccreditedIndividualsController < ApplicationController
+      service_tag 'representation-management'
+      skip_before_action :authenticate
+      before_action :feature_enabled
+      before_action :verify_type
+      before_action :verify_sort
+      before_action :verify_long
+      before_action :verify_lat
+      before_action :verify_distance
+
+      DEFAULT_PAGE = 1
+      DEFAULT_PER_PAGE = 10
+      DEFAULT_SORT = 'distance_asc'
+      PERMITTED_MAX_DISTANCES = [5, 10, 25, 50, 100, 200].freeze # in miles, no distance provided will default to "all"
+      PERMITTED_SORTS = %w[distance_asc first_name_asc first_name_desc last_name_asc last_name_desc].freeze
+      PERMITTED_TYPES = %w[attorney claims_agent representative].freeze
+
+      def index
+        collection = Common::Collection.new(AccreditedIndividual, data: individual_query)
+        resource = collection.paginate(**pagination_params)
+
+        render json: serializer_class.new(resource.data, { meta: resource.metadata })
```

Comment:
Why not just call `RepresentationManagement::AccreditedIndividuals::IndividualSerializer.new(resource.data, { meta: resource.metadata })` and delete `#serializer_class`? I'm wondering if this is left over from the code in the `veterans` module.

Reply:
The class name is too long and it's a bit cleaner to keep it on one line. I can refactor to not need a private method. It's almost always preferable to avoid disabling rubocop rules.

Reply:
Yes, I agree re not disabling cops if you don't have to.

Here are a couple of ideas (whatever you prefer though):

```
          render json: RepresentationManagement::AccreditedIndividuals::IndividualSerializer.new(
            resource.data,
            { meta: resource.metadata }
          )
```

Or

```
          render(
            json: RepresentationManagement::AccreditedIndividuals::IndividualSerializer.new(
              resource.data,
              { meta: resource.metadata }
            )
          )
 ```

Reply:
@jvcAdHoc here's another idea:

```ruby
def index
  collection = Common::Collection.new(AccreditedIndividual, data: individual_query)
  resource = collection.paginate(**pagination_params)
  options = { meta: resource.metadata }
  data = resource.data
  
  render json: RepresentationManagement::AccreditedIndividuals::IndividualSerializer.new(data, options)
end
```